webcompat/views.py

```diff
@@ -306,3 +321,13 @@ def contributors():
 def cssfixme():
     '''Route for CSS Fix me tool'''
     return render_template('cssfixme.html')
+
+
+@app.route('/csp-report', methods=['POST'])
+def log_report():
+    '''Route to record CSP header violations.'''
+    if 'application/csp-report' not in request.headers.get('content-type'):
+        return ('Wrong Content-Type.', 400)
+    with open(app.config['CSP_REPORTS_LOG'], 'a') as r:
+        r.write(request.data + '\n')
```

Comment:
Another reason to do that is with the combination of a logging file configuration to allow for activating and deactivating the feature when needed and have timestamp when the log is happening. 

```python
CSP_LOG = True # or False
if CSP_LOG:
    csp_log.info(request.data)
```